rundeckapp/grails-spa/packages/ui-trellis/src/app/components/home/HomeActionsMenu.vue

```diff
@@ -0,0 +1,73 @@
+<template>
+  <dropdown menu-right append-to-body ref="dropdown">
+    <a role="button" class="as-block link-hover link-quiet link-block-padded text-inverse dropdown-toggle">
```

Comment:
Maybe changing this to a `<btn type="link">` would be better? currently i can't tab to this drop down toggle for some reason.